### ISSUE TITLE
Small bugs in kernel update error screen

### ISSUE THREAD

**Issue by Caseyb87** [spyder-ide/spyder#17425]
Issue is raised in #17353 with updating spyder-kernel in custom environments.

Putting aside the fact that Anaconda doesn't have that version of spyder-kernels yet... there are still two small bugs here on the error screen. 

Firstly, if the required version is 2.2.1, then the error should say 2.2.1 needs installing (not 2.2). Installing 2.2 as suggested still results in the same error which coud be confusing for the user.

Secondly the dash in ``conda install spyder‑kernels=2.2`` doesn't copy in to a Windows terminal correctly (displays a question mark inside a box). A font issue?

**Comment by ccordoba12** [comment]
Hey @Caseyb87, thanks for reporting. You said:

> Firstly, if the required version is 2.2.1, then the error should say 2.2.1 needs installing (not 2.2). Installing 2.2 as suggested still results in the same error which coud be confusing for the user.

It clearly says so (look at the third line):

![image](https://user-images.githubusercontent.com/88217916/156491010-de81edd9-361c-4f20-a9d7-fd5496c1913f.png)

I know the command is

    `conda install spyder‑kernels=2.2`, not

    `conda install spyder‑kernels=2.2.1`

as you suggest, but that's necessary to install any `2.2` version (i.e. 2.2.0, 2.2.1, 2.2.2, etc). compatible with the current Spyder one. In this case, `2.2.0` should be compatible with Spyder 5.2.2, but we had to increase that requirement to fix some serious bug with Matplotlib.

At the end, I'd say this problem has more to do with Anaconda than Spyder, because it takes them a really long time to update Spyder and spyder-kernels. So `2.2.0` is already available, but then `2.2.1` was released more than a month ago and it hasn't been uploaded yet. We know that's a huge inconvenience for our users, but we can't do anything about it because it's up to them really.

> Secondly the dash in conda install spyder‑kernels=2.2 doesn't copy in to a Windows terminal correctly (displays a question mark inside a box). A font issue?

No, that's an escaped html dash (not the one you get with your keyboard), so we can display correctly in that message. I didn't know people was copy/pasting directly to their console. We'll see what we can do about it.
